rpm/sofia-sip.spec

```diff
@@ -86,9 +86,7 @@ find %{buildroot} -name \*.h.in -delete
 %postun glib -p /sbin/ldconfig
 
 %files
-%defattr(-,root,root,-)
```

Comment:
If you drop `%defattr` here then it's better to remove such lines in other sub-packages.

Reply:
Brought it back